app/assets/stylesheets/application.scss

```diff
@@ -1 +1,21 @@
 @import "bootstrap";
+
```

Comment:
Please follow this styleguid: https://netguru.atlassian.net/wiki/spaces/FRONT/pages/75825767/CSS+Styleguide